devtools/mkcommit.c

```diff
@@ -388,26 +404,31 @@ int main(int argc, char *argv[])
 				&local_per_commit_point, commitnum, LOCAL);
 
 	printf("## local_commitment\n"
-	       "# input amount %s, funding_wscript %s, key %s\n",
+	       "# input amount %s, funding_wscript %s, pubkey %s\n",
 	       type_to_string(NULL, struct amount_sat, &funding_amount),
 	       tal_hex(NULL, funding_wscript),
 	       type_to_string(NULL, struct pubkey, &funding_localkey));
 	printf("# unsigned local commitment tx: %s\n",
 	       tal_hex(NULL, linearize_tx(NULL, local_txs[0])));
 
+	/* Get the hash out, for printing */
+	bitcoin_tx_hash_for_sig(local_txs[0], 0, funding_wscript,
+				SIGHASH_ALL, &hash);
 	sign_tx_input(local_txs[0], 0, NULL, funding_wscript,
 		      &local.funding_privkey,
 		      &funding_localkey,
 		      SIGHASH_ALL,
 		      &local_sig);
 	printf("localsig_on_local: %s\n", sig_as_hex(&local_sig));
+	print_privkey_and_hash(&hash, &local.funding_privkey);
```

Comment:
I would get rid of the sig_as_hex and just use SIG() notation for that?  Maybe just:
```
printf("localsig_on_local: %s\n", sig_notation(&hash, local.funding_privkey));
```